### DISCUSSION TITLE
Separate `:rename` command that moves within the current directory

### DISCUSSION THREAD

**Discussion by dob9601** [helix-editor/helix#11053] [Ideas] [answered]
I work on projects that unfortunately have some deeply nested folder structures. It would be helpful if there was a command along the lines of `:rename` which moves the file within the current directory as opposed to having to specify the full path every single time.

I would be happy to create a pull request myself but thought I'd throw up an issue first to see if its something the project would be interested in accepting

**Comment by the-mikedavis** [comment, marked as answer]
You can use the document path register `%` instead. In the `:move` prompt use `<C-r>%` to insert the current document's path and then use `C-w` to delete the last segment of the path

**Reply by dob9601** [reply]
Good to know, thanks!
